index.html

```diff
@@ -92,19 +161,44 @@ <h3 id="the-difficulties-of-ruby-processing">The Difficulties of Ruby Processing
     whether it can be allowed to be laid over the characters preceding or following,
     and whether this affects the position of the base characters</p>
     <aside class=note title="Protrusion of ruby from base characters">
-      In movable type typesetting,
-      ruby characters protruding from their base characters
-      may not hang over neighboring kanji,
-      but often were allowed to hang over neighboring kana.
-      However, when the ruby is katakana,
-      some publishers would set it so that
-      it would not hang over the kana neighboring the base character.
-      Also, when the characters around the base characters
-      were kanji on one side and kana on the other side,
-      for the sake of balance
-      the ruby would sometimes be set so as to hang
-      neither over the kanji nor over the kana
-      (which would therefore both be spaced away from the base character).
+      <span class="checkme">
+      From principle not to have extra spacing in Japanese composition, 
+      one method of layout is to lay over the ruby string preceding or 
+      following Kana characters but not to lay over the ruby string proceding 
+      or following Kanji characters. 
+      With this method, processing may go smoothly for the case with Kana 
+      characters as both proceding and following of ruby base text, 
+      but there is possible case of unbalanced display, 
+      like a combination of Kana character as proceding of ruby base text 
+      and Kanji character as following of ruby base text
+      (see <a href="#ruby-protruding-mono"></a> as mono-ruby case, 
+      upper panel takes a rule to lay over the ruby string preceding or
```

Comment:
The "upper panel" (and "lower panel" below) doesn't sound very natural. I think it refers to the upper example in Figure 1 (and the second example in Figure 3 for the "lower panel" below) Maybe just use "first example" and "second example"?

Reply:
modified.